### ISSUE TITLE
Running tensorflow-gpu on windows 10

### ISSUE THREAD

**Issue by Hasan-ta** [tensorflow/tensorflow#7195]
### What related GitHub issues or StackOverflow threads have you found by searching the web for your problem?
I am trying to run to train a convolutional network on lenet dataset but whenever I open a session and start `sess.run(training_operation,` feed_dict={x: batch_x, y: batch_y})` I get the following error:

> name: GeForce GTX 980
> major: 5 minor: 2 memoryClockRate (GHz) 1.178
> pciBusID 0000:01:00.0
> Total memory: 8.00GiB
> Free memory: 6.70GiB
> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\common_runtime\gpu\gpu_device.cc:906] DMA: 0
> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\common_runtime\gpu\gpu_device.cc:916] 0:   Y
> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\common_runtime\gpu\gpu_device.cc:975] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX 980, pci bus id: 0000:01:00.0)
> E c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\cuda\cuda_event.cc:49] Error polling for event status: failed to query event: CUDA_ERROR_LAUNCH_FAILED
> F c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\core\common_runtime\gpu\gpu_event_mgr.cc:198] Unexpected Event status: 1
> E c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\cuda\cuda_dnn.cc:385] could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR

### Environment info
Operating System: Windows 10 Home

Installed version of CUDA and cuDNN:  
(please attach the output of `ls -l /path/to/cuda/lib/libcud*`):
CUDA 8.0
cuDNN 5.1

If installed from binary pip package, provide:

1. A link to the pip package you installed:

> pip install --upgrade https://storage.googleapis.com/tensorflow/windows/gpu/tensorflow_gpu-0.12.1-cp35-cp35m-win_amd64.whl

2. The output from `python -c "import tensorflow; print(tensorflow.__version__)"`.

> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:128] successfully opened CUDA library cublas64_80.dll locally
> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:128] successfully opened CUDA library cudnn64_5.dll locally
> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:128] successfully opened CUDA library cufft64_80.dll locally
> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:128] successfully opened CUDA library nvcuda.dll locally
> I c:\tf_jenkins\home\workspace\release-win\device\gpu\os\windows\tensorflow\stream_executor\dso_loader.cc:128] successfully opened CUDA library curand64_80.dll locally
> 0.12.1

### Logs or other output that would be helpful
(If logs are large, please upload as attachment or provide link).

**Comment by mrry** [comment]
Can you try upgrading to the 1.0.0rc version of `tensorflow-gpu`? There were some buggy GPU kernels in the 0.12 release that caused similar failures.

**Comment by MIC7** [comment]
I have a similar setup, but with a gtx-1080, and was having the same problem. I did indeed get it working by using the 1.0.0rc version of tensorflow-gpu. It did require slight tweaking of syntax though, so that I had to explicitly add "labels=" and "logits=" for softmax_cross_entropy_with_logits.

**Comment by gunan** [comment]
Looks like the problem is resolved in a more recent version.
Closing the issue.

**Comment by aghasd5** [comment]
@mrry 
Can you try upgrading to the 1.0.0rc version of tensorflow-gpu? There were some buggy GPU kernels in the 0.12 release that caused similar failures.

from where may i install 1.0.0rc version of tensorflow-gpu.

**Comment by gunan** [comment]
I assume you are asking us to update our binaries uploaded to pypi.
We will update those once we verify that rc(release candidate) versions are sufficiently stable.
Until then, please Use the URLs here to download our pre-built binaries for different operating systems:
https://www.tensorflow.org/versions/r1.0/get_started/os_setup#pip_installation
